moler/runner.py

```diff
@@ -532,12 +534,27 @@ def _feed_loop(self, connection_observer, stop_feeding, observer_lock):
                             connection_observer.in_terminating = True
                         else:
                             break
+            else:
+                self._call_on_inactivity(connection_observer=connection_observer, current_time=current_time)
 
             if self._in_shutdown:
                 self.logger.debug("shutdown so cancelling {}".format(connection_observer))
                 connection_observer.cancel()
             time.sleep(self._tick)  # give moler_conn a chance to feed observer
 
+    def _call_on_inactivity(self, connection_observer, current_time):
+        """
+        Call on_inactivity on connection_observer if needed.
+
+        :param connection_observer: ConnectionObserver object.
+        :param current_time: current time in seconds.
+        :return: None
+        """
+        if connection_observer.inactivity_timeout > 0.0 and connection_observer.last_feed_time is not None:
```

Comment:
add braces in expression
```python
if (connection_observer.inactivity_timeout > 0.0) and (connection_observer.last_feed_time is not None):
```

Reply:
This won't work in "initial silence" case. I mean:
1. we send data over connection
2. no data comes within connection_observer.inactivity_timeout
This is so because connection_observer.last_feed_time is set (for the first time) inside `secure_data_received` means when data comes on connection.

Question is if we want to handle this "initial silence" or the only thing we care is silence after data has been received - means: indication of some failure.

Reply:
Added to method submit in runner.